components/bl-leafletMap-component/component.json

```diff
@@ -0,0 +1,78 @@
+{
+  "id": "c_fd8c99a633b33a236df39c4b2a76f8c2",
+  "name": "Leaflet map",
+  "description": "The component allows you to work with the map.",
+  "showInToolbox": true,
+  "faIcon": "map",
+  "mainJS": "dist/index.js",
+  "type": "custom",
+  "category": "Custom Components",
+  "properties": [
```

Comment:
Please, take a look at our Backendless Google Map component logic, and add handlers which match API of this library.

Reply:
I added a click event, others could not be implemented due to the features of the library.